### DISCUSSION TITLE
Where is the "Anti-aliasing" module or function?

### DISCUSSION THREAD

**Discussion by c61292558** [fltk-rs/fltk-rs#1365] [Q&A] [answered]
As title,
where is the " Anti-aliasing " module or function ?

Thanks for your help.

**Comment by MoAlyousef** [comment, marked as answer]
There is an enums::Mode::MultiSample which you can pass to an Opengl window.set_mode(). Otherwise drawing on all oses (on normal windows) uses antialiasing. You can disable it by using no-pango on linux and no-gdiplus on windows.

**Reply by c61292558** [reply]
What is “No-pango”？I use Linux system.How to set ?  
Besides,and how can I draw OpenGL shapes with fltk-rs ?

**Reply by MoAlyousef** [reply]
Please check the fltk-rs/demos repo which contains several OpenGL examples (using raw opengl, glium, glow and others).

**Reply by c61292558** [reply]
Thank you ,and how to make the fonts "Anti-aliasing"in DoubleWindow(not GlutWindow)? Like "@<-","@circle","@square" etc. on the label .

**Reply by MoAlyousef** [reply]
FLTK does anti-aliased drawing by default.

**Reply by c61292558** [reply]
Oh,great .I understand by your help. Thank you very much.
